src/storage/txn/actions/prewrite.rs

```diff
@@ -408,6 +437,14 @@ impl<'a> PrewriteMutation<'a> {
             return Ok(LockStatus::Pessimistic(lock.for_update_ts));
         }
 
+        if generation_to_write > 0 && lock.generation >= generation_to_write {
```

Comment:
Can we skip this key when `lock.generation == generation_to_write`, imaging this case:

1. flush success but the response1 is slow.
2. retried flush got generation-out-of-order error.

In this case, client know the retried flush request is failed, but nothing can do except waiting for response of the first flush request, if this response is lost, the transaction has to be fail.

Reply:
Let's simply skip all mutations with `GenerationOutOfOrder` errors.